deepchem/models/tensorgraph/layers.py

```diff
@@ -918,6 +977,30 @@ def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
     return out_tensor
 
 
+class StopGradient(Layer):
+  """Block the flow of gradients.
+
+  This layer copies its input directly to its output, but reports that all
+  gradients of its output are zero.  This means, for example, that optimizers
+  will not try to optimize anything "upstream" of this layer."""
```

Comment:
Could you add a simple usage example here in docstring?

Reply:
Is the proposal to initially put a no-op layer in the tensorgraph and then on copying it to replace it with a StopGradient?

Reply:
Nevermind you would add this layer after you did a copy with an input being the just copied layers that you want to "freeze".